scripts/staging/validate.sh

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+SCRIPT_DIR="$(dirname "$(realpath "${BASH_SOURCE[0]:-"$0"}")")"
+ROOT_DIR="${SCRIPT_DIR}/../.."
+
+: "${PARENT_ROOT_DIR:=$ROOT_DIR}"
+
+source "$ROOT_DIR/mayastor/scripts/utils/log.sh"
+NO_RUN=true . "$PARENT_ROOT_DIR/scripts/release.sh"
+
+DOCKERHUB_ORG="${DOCKERHUB_ORG:-openebs}"
+GITHUB_ORG="${GITHUB_ORG:-openebs}"
+IMAGE_REGISTRY="${IMAGE_REGISTRY:-docker.io}"
+CHART_REGISTRY="${CHART_REGISTRY:-gh-pages}"
+CHART_NAME="${CHART_NAME:-openebs}"
+NAMESPACE="${NAMESPACE:-${GITHUB_ORG}/helm}"
+INDEX_REMOTE="${INDEX_REMOTE:-origin}"
+INDEX_BRANCH="${INDEX_BRANCH:-gh-pages}"
+INDEX_BRANCH_FILE="${INDEX_BRANCH_FILE:-index.yaml}"
+
+CHART_FILE="${CHART_FILE:-${PARENT_ROOT_DIR}/charts/Chart.yaml}"
+
+TRIGGER=""
+TAG=""
+SKIP_INPUT_VALIDATION=false
+SKIP_ARTIFACT_CHECK=false
+
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --trigger|--type)
+            TRIGGER="$2"
+            shift 2
+            ;;
+        --tag)
+            TAG="$2"
+            shift 2
+            ;;
+        --skip-input-validation)
+            SKIP_INPUT_VALIDATION=true
+            shift
+            ;;
+        --skip-artifact-check)
+            SKIP_ARTIFACT_CHECK=true
+            shift
+            ;;
+        -h|--help)
+            cat <<EOF
+Usage: $0 --trigger <trigger> [--tag <tag>] [options]
+
+Options:
+  --trigger <type>             Trigger type: release, staging, develop, or prerelease
+  --type <type>                Alias for --trigger (backward compatibility)
+  --tag <tag>                  Release tag (e.g., v2.9.0)
+  --skip-input-validation      Skip input validation checks
+  --skip-artifact-check        Skip artifact existence checks
+  -h, --help                   Show this help message
+EOF
+            exit 0
+            ;;
+        *)
+            log_fatal "Unknown option $1"
+            ;;
+    esac
+done
+
+if [[ -z "$TAG" ]]; then
+    if [[ ! -f "$CHART_FILE" ]]; then
+        log_fatal "❌ Error: Chart.yaml not found at ${CHART_FILE} and no tag provided"
+    fi
+    VERSION=$(awk -F': ' '/^version:/ {print $2}' "$CHART_FILE" | tr -d ' ')
+    TAG="v${VERSION}"
+    echo "Using chart version from $CHART_FILE: $TAG"
+fi
+
+if [[ "$SKIP_INPUT_VALIDATION" != true ]]; then
+    echo "Validating trigger: $TRIGGER"
+    echo "Validating tag: $TAG"
+    
+    case "$TRIGGER" in
+        "release"|"staging"|"develop"|"prerelease")
```

Comment:
Quotes not needed here right?

```suggestion
        release|staging|develop|prerelease)
```